docs/changelog.rst

```diff
@@ -1,6 +1,37 @@
 Release Notes
 =============
 
+0.7
+---
+
+Data access
+~~~~~~~~~~~
+
+- Trying to open a run directory to which you don't have read access now
+  correctly raises PermissionError (:ghpull:`210`).
+- :func:`~.stack_detector_data` has a new parameter ``real_array``. Passing
+  ``real_array=False`` avoids copying the data into a temporary array on the way
+  to assembling images with detector geometry (:ghpull:`196`).
```

Comment:
Is this explained further in the documentation? Should we point to the documentation if so? (I didn't find it clear just from the statement here.)

Reply:
A bit - the `:func:` reference will link to the [function documentation](https://karabo-data.readthedocs.io/en/latest/agipd_lpd_data.html#karabo_data.stack_detector_data).

It's a tricky thing to explain clearly, but the key points are:

- Passing `real_array=False` is faster if it works
- It might not work everywhere that `real_array=True` does (so True is the default)
- But it should work for stacking detector data.